src/api/api.ts

```diff
@@ -0,0 +1,36 @@
+import { Comment } from '../types/Comment';
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>('/users');
```

Comment:
Consider adding error handling for the API request to manage potential failures and provide feedback to the user.